docs/dev/features/object-storage/blobcsi.md

```diff
@@ -125,6 +125,139 @@ ACL.
 `subfolder` if a project requires sub-folder mounting then this folder must be created before provisioning the volumes. If sub-folder is not required, leave the name as empty string
 5. AAW will add the SP to the appropriate AD Group. Depending on the container created and environment the SP will be added to either: `AAW-PROD-External-Unclassified-SPs`, `AAW-PROD-Internal-Unclassified-SPs`, `AAW-PROD-External-Protected-B-SPs`, `AAW-PROD-Internal-Unclassified-SPs`
 Note: For debugging there are test containers created in all Prod and Dev storage accounts for future development and testing.
+
```

Comment:
Is this just a much more detailed, step-by-step walk through of the section above it? `How to create a new FDI container?`

Reply:
Yes, it's a more detailed, step-by-step version that I created when doing the work.